apps/vanilla/src/pages/index.html

```diff
@@ -1,16 +1,138 @@
+                    <li class="book-card">
+                        <img class="book-card__img" src="./assets/images/04.png" alt="Refactoring UI book" />
+                        <p class="text-body-sm">
+                            7 Навыков высокэффективных...
+                        </p>
+                    </li>
+                </ul>
+            </section>
+        </article>
+        <!-- Aside filters -->
+        <aside class="catalog__filter">
+            <div class="fixed-container">
+                <h2 class="text-subtitle">filters</h2>
+                <form class="filter-form">
+                    <div class="filter-form__search">
+                        <i>
+                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M21 21L15.803 15.803M15.803 15.803C17.2096 14.3965 17.9998 12.4887 17.9998 10.4995C17.9998 8.51035 17.2096 6.60262 15.803 5.19605C14.3965 3.78947 12.4887 2.99927 10.4995 2.99927C8.51035 2.99927 6.60262 3.78947 5.19605 5.19605C3.78947 6.60262 2.99927 8.51035 2.99927 10.4995C2.99927 12.4887 3.78947 14.3965 5.19605 15.803C6.60262 17.2096 8.51035 17.9998 10.4995 17.9998C12.4887 17.9998 14.3965 17.2096 15.803 15.803Z" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" />
+                            </svg>
+                        </i>
+                        <input class="text-body" type="search" name="searchInput" placeholder="e.g. “Refactoring”">
+                    </div>
+                    <div class="filter-form__select">
+                        <i>
+                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path d="M19.5 8.25L12 15.75L4.5 8.25" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" />
+                            </svg>
+                        </i>
+                        <select name="filterSelect" class="filter-form__select text-body">
+                            <option value="">Choose sort field</option>
+                        </select>
+                    </div>
+                    <label class="filter-form__checkbox checkbox">
+                        <input class="checkbox__input" type="checkbox" name="checkboxField">
+                        <span class="checkbox__mark"></span>
+                        <p class="text-body">Show comics</p>
+                    </label>
+                </form>
+            </div>
+        </aside>
+    </main>
+    <!-- Footer -->
+    <footer class="books-footer">
+        <section class="books-footer__info">
+            <div class="books-footer__additional">
+                <p class="text-logo">Books app</p>
+                <p class="text-body">© 2023</p>
+            </div>
+            <a class="footer-link text-body" href="/">Github</a>
+            </div>
```

Comment:
Try to avoid invalid html.
```suggestion
            ̶<̶/̶d̶i̶v̶>̶
```